comments/migrations/0003_comment_comment_statut_idx.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 4.2.7 on 2023-12-06 14:06
+
+from django.db import migrations, models
+from django.contrib.postgres.operations import AddIndexConcurrently
+
+
+class Migration(migrations.Migration):
+    atomic = False
+
+    dependencies = [
+        ("comments", "0002_alter_comment_uuid_objet"),
+    ]
+
+    operations = [
+        AddIndexConcurrently(
```

Comment:
This tricks allows to create the new index without any lock.
The django doc is [here](https://docs.djangoproject.com/fr/4.2/ref/contrib/postgres/operations/#concurrent-index-operations) ;-)